src/opm/parser/eclipse/share/keywords/keyword_list.cmake

```diff
@@ -9,6 +9,25 @@
 # you are therefore encouraged to go to the opm-simulators repository and update
 # the file: opm/autodiff/MissingFeatures.cpp
 
+#Some keywords are found to be of 'special' structure:
+#W             WHAT
+#ECONT         Terminates with two /
+#COALNUM       matrix structure
+#COMPDATM      alias for COMPDATL
+#COMPINJK      undef default
```

Comment:
This looks clear enough to me - they all have default value 0 - the "Allow any value" functionality will be handeled in the subsequent C++ layer.